### PR TITLE
hs sandbox create

### PR DESCRIPTION
**Pull request by adamawang**
## Description and Context

Updates for `hs sandbox create`
- The CLI auto prompts the user to generate a PAK and add to the config file. 
- The sandbox name is pulled in by default, but gives the user an option to use a custom one. 
- Adds an `account` option to the PAK prompt, where we forward the user to the PAK generation page rather than the account picker. 
- Adds an error message specific to missing scopes when creating a sandbox

## Screenshots

Screen Shot 2022-05-06 at 5 06 47 PM

Screen Shot 2022-05-06 at 5 03 28 PM

## TODO

Have some updates to `hs init` and `hs auth` that I will put up in a separate PR. Those updates will also use the `accountId` option in the PAK prompt

### CODE REVIEW

File: packages/cli-lib/lang/en.lyaml
Comment (adamawang) [inline packages/cli-lib/lang/en.lyaml:556]: This is the same line we use in the init step. I currently have it prompt the user for a name, but defaults to the sandbox name the user just typed in in case they want it custom. 

Keeping Mark's [comment in mind here](https://hubspot.slack.com/archives/C01GV708YH3/p1651516531083669?thread_ts=1651167455.710479&cid=C01GV708YH3), do we want to skip the prompt and just set the same name by default?

File: packages/cli/commands/sandbox/create.js
Comment (adamawang) [inline packages/cli/commands/sandbox/create.js:140]: Added a scope specific error here:

Screen Shot 2022-05-06 at 2 55 46 PM

File: packages/cli-lib/lang/en.lyaml
Comment (brandenrodgers) [reply packages/cli-lib/lang/en.lyaml:556]: That would be really cool! IMO what you have now is reasonable, and we can always tack that on as an update later if we want to.

File: packages/cli/commands/sandbox/create.js
Comment (brandenrodgers) [inline packages/cli/commands/sandbox/create.js:73]: Don't worry about handling this in this PR, but I'm noticing that we're accumulating some repetitive auth-related prompts (I have some WIP work that's also adding more of it). I may try to consolidate some of it later on.

File: packages/cli/commands/sandbox/create.js
Comment (brandenrodgers) [inline packages/cli/commands/sandbox/create.js:134]: Can this also be added to the en.lyaml?

File: packages/cli/commands/sandbox/create.js
Comment (brandenrodgers) [reply packages/cli/commands/sandbox/create.js:144]: Looks like we have a util for this [here](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/errorHandlers/apiErrors.js#L24). You may be able to export that and use it here

File: packages/cli/commands/sandbox/create.js
Comment (brandenrodgers) [inline packages/cli/commands/sandbox/create.js:148]: We may want to get some design eyes on this message.

File: packages/cli/commands/sandbox/create.js
Comment (adamawang) [reply packages/cli/commands/sandbox/create.js:134]: Ah yep! I originally had it in `cli-lib` but ended up moving it back to the cli package. Will add translations

File: packages/cli-lib/lang/en.lyaml
Comment (adamawang) [inline packages/cli-lib/lang/en.lyaml:576]: Not sure the best way to do this, but I added a bunch of white space here so the messaging wouldn't get bunched up/all on one line

Screen Shot 2022-05-06 at 4 22 12 PM

Comment (brandenrodgers) [reply packages/cli-lib/lang/en.lyaml:576]: Nice 👍  I like it with the extra spacing. FWIW I've been using blank `logger.log` calls to add newlines. You can also use `\n` in the en.lyaml file to add newlines (there should be examples of it in there somewhere).

**Comment by adamawang** [comment]
Received updated copy and direction from Mark - updated screenshots in main comment!

File: packages/cli/lib/prompts/personalAccessKeyPrompt.js
Comment (brandenrodgers) [inline packages/cli/lib/prompts/personalAccessKeyPrompt.js:18]: I'm sure this is already on your radar, but just a reminder to update things accordingly after [this](https://github.com/HubSpot/hubspot-cli/pull/691) is merged.

Comment (adamawang) [reply packages/cli/lib/prompts/personalAccessKeyPrompt.js]: Good catch - updated and resolved! Waiting on Erin if she has any updates before merging, thread here: https://hubspot.slack.com/archives/C01GV708YH3/p1652106906292789?thread_ts=1651863988.648859&cid=C01GV708YH3

**Review by brandenrodgers — APPROVED** [review]
LGTM (pending final copy updates)

File: packages/cli-lib/lang/en.lyaml
Comment (erincouse) [reply packages/cli-lib/lang/en.lyaml:561]: I think we should set the same name by default, 100%. Let's make everyone's lives easier.

File: packages/cli/commands/sandbox/create.js
Comment (erincouse) [reply packages/cli/commands/sandbox/create.js:153]: what's the message? Forgive my Github incompetence.

File: packages/cli/commands/sandbox/create.js
Comment (erincouse) [inline packages/cli/commands/sandbox/create.js]: is this the error message? I would say "The personal access key you provided doesn't give access to sandbox permissions. Deactivate the key and generate a new one with the correct permissions. After that, run `hs auth` again and enter the new key."

File: packages/cli/commands/sandbox/create.js
Comment (erincouse) [inline packages/cli/commands/sandbox/create.js]: are you authenticating with a sandbox, or are you authenticating the sandbox? genuine question, not sure if there's nuance needed with the language here

File: packages/cli-lib/lang/en.lyaml
Comment (erincouse) [inline packages/cli-lib/lang/en.lyaml]: do they need to take an action to get the the following step?

File: packages/cli/commands/sandbox/create.js
Comment (adamawang) [reply packages/cli/commands/sandbox/create.js]: Yep this is a permissions error message here - I'll get this change rolled in!

File: packages/cli/commands/sandbox/create.js
Comment (adamawang) [reply packages/cli/commands/sandbox/create.js]: This step was removed in a later commit here so we should be all set on this

File: packages/cli-lib/lang/en.lyaml
Comment (adamawang) [reply packages/cli-lib/lang/en.lyaml]: Same as above - addressed in a later commit and this section has been removed

File: packages/cli/commands/sandbox/create.js
Comment (adamawang) [reply packages/cli/commands/sandbox/create.js]: After checking in with Erin, we adjusted to use `The personal access key you provided doesn't include sandbox permissions` in place of the "ERROR" line. Also adjusted `Visit` to `Go to` for the URL line.

**Pull request merged**
